README.md

```diff
@@ -132,11 +132,9 @@ Bamboo offers `deliver_later` on your mailers to send emails in the background s
 By default delivering later uses [`Bamboo.TaskSupervisorStrategy`](https://hexdocs.pm/bamboo/Bamboo.TaskSupervisorStrategy.html). This strategy sends the email right away, but does so in the background without linking to the calling process, so errors in the mailer won't bring down your app.
 
 If you need something more custom you can
-create a strategy with [Bamboo.DeliverLaterStrategy](https://hex.pm/packages/bamboo). For example, you could create strategies
+create a strategy with [`Bamboo.DeliverLaterStrategy`](https://hexdocs.pm/bamboo/Bamboo.DeliverLaterStrategy.html). For example, you could create strategies
```

Comment:
You can remove the `(link)` part of this and hex will automatically detect that there is a module called `Bamboo.DeliverLaterStrategy` and it will link to the module docs to that module. You can run `mix docs` then `open doc/index.html`, navigate to the docs and you should see that it is linked automatically. Does that make sense?